### DISCUSSION TITLE
十堰穿透无法启动

### DISCUSSION THREAD

**Discussion by CloudyEagle** [natfrp/wiki#128] [使用问题] [answered]
我穿透的是Windows远程桌面，今天一天都连不上，
前面参看了一下帮助文档，我这ping也ping不通，
是节点问题还是我的网络问题嘞，需要重启电脑？
![image](https://user-images.githubusercontent.com/58777085/182864238-c958941c-9ca1-47b5-a964-de2fc5861bee.png)
![image](https://user-images.githubusercontent.com/58777085/182864121-436cd3bc-2dec-48b2-95ef-d096eac4f3d5.png)

**Comment by endcreeper861** [comment, marked as answer]
十堰电信最近受到了大量攻击，暂时不能用了
![image](https://user-images.githubusercontent.com/94234877/182891508-4f19e5ab-9b85-47cc-8c97-0b6bc0ba676b.png)

**Comment by whao123yahoo** [comment]
还没修复吗
